src/Strategy/AbstractStrategy.php

```diff
@@ -41,7 +41,7 @@ public function getContainer()
      *
      * @param  string|array|\Closure $controller
      * @param  array                 $vars
-     * @return \League\Http\ResponseInterface
+     * @return Response
```

Comment:
Can this be FQN please. I'm aware it doesn't need to be but the rest of the package is.